docs/en/sql-reference/functions/url-functions.md

```diff
@@ -500,3 +500,39 @@ Result:
 │ http://bigmir.net/?c=d&e=f#g │ http://bigmir.net/?a=b#g │
 └──────────────────────────────┴──────────────────────────┘
 ```
+
+### resolveRelativeURL(RELATIVE_URL, BASE_URL)
+
+The resolveRelativeURL function resolves a relative URL against a base URL and returns the absolute URL. 
+**Syntax**
+
+``` sql
+resolveRelativeURL(relative_url, base_url)
+```
+
+**Arguments**
+
+- `relative_url` — the name of the relative URL. [String](../../sql-reference/data-types/string.md).
+- `base_url` — the name of the base URL. [String](../../sql-reference/data-types/string.md) or [Array](../../sql-reference/data-types/array.md) of Strings.
```

Comment:
Why and how we can use array as an argument?

Reply:
I wrote this comment before checking a code, so I suppose we  can just remove mention `Array`. If it accepted `Array` it should be like `resolveRelativeURL('aaa', ['a','b','c'])`, but function supports only `String`